### PR TITLE
Fix/1278 build zip

### LINKED ISSUE #1278: Update for WooRelease
- The build script needs to generate woocommerce-gateway-stripe.zip

### PR DESCRIPTION
**Pull request by allendav**
Fixes #1278  .

#### Changes proposed in this Pull Request:
- Adds ZIP file generation

#### To Test:
- `npm run build`
- Verify woocommerce-gateway-stripe.zip is created in the root of the repo
- Install woocommerce-gateway-stripe.zip on a site
- Ensure you can activate the plugin successfully

-------------------
- [ ] Make sure your changes respect [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- [ ] Did you make changes, or create a **new .js file**? If **Gruntfile.js** exists in the repo, make sure to run `grunt`.

### CODE REVIEW

File: tasks/release.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
+/* eslint-disable no-console */
+/* eslint no-process-exit: 0, no-undef: 0, strict: 0 */
+'use strict';
+require( 'shelljs/global' );
+const colors = require( 'colors' );
```
Comment (luizreis) [inline tasks/release.js:5]: When I tried to run this for the first time I got the error `Error: Cannot find module 'colors'` and had to install `colors` using `npm` to be able to run it, so we should probably add it to `package.json` too.

File: package.json [hunk 2]
```diff
@@ -42,7 +44,7 @@
     }
   },
   "scripts": {
-    "build": "npm run uglify && npm run makepot && npm run sass",
+    "build": "npm install && composer install && npm run uglify && npm run makepot && npm run sass && node tasks/release.js && mv release/woocommerce-gateway-stripe.zip .",
```
Comment (luizreis) [inline package.json:47]: What do you think of splitting this to improve readability? Something along the lines of:

```suggestion
    "prebuild": "npm install && composer install",
    "build": "npm run uglify && npm run makepot && npm run sass && npm run build:release",
    "build:release": "node tasks/release.js && mv release/woocommerce-gateway-stripe.zip .",
```

File: tasks/release.js (hunk 1 above)
Comment (allendav) [reply tasks/release.js:5]: Fixed in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1279/commits/e2657601e3bb1ce6786e5a1b93f60b7ce92a7162

**Comment by allendav** [comment]
Almost there.  Might need one more tweak for woorelease

wr simulate https://github.com/woocommerce/woocommerce-gateway-stripe/tree/fix/1278-build-zip

fails with (I'm attempting to do the command line that doesn't require product version, so the ERROR is odd)

```
WR [00:11:38] [NOTICE] Completed simulated deploy to WPorg.
WR [00:11:38] [ERROR] The "--product_version" option requires a value.
WR [00:11:38] [ERROR] Woorelease finished with errors! Please correct the error and retry.
```

**Comment by allendav** [comment]
FYI - looks like woorelease needs a tweak, so we are moving forward with this and will release 4.5.1 manually

**Pull request merged**